mantidproject/mantid
# Add WorkspaceIndex attribute in InelasticDiffSphere and InelasticDiffRotDiscreteCircle

Fixes [#11447](http://trac.mantidproject.org/mantid/ticket/11447).

To test:
- Load `irs26176_graphite002_red.nxs`
- Run the following script with the log set to debug and notice there is log output showing the value of Q used for each spectrum, this value should be equal to that shown in Show Detectors
- Add the Q parameter in (add `,Q=1.2` after the Shift parameter), run again, notice now the same Q value is used for all spectra (as was done before).
- Do the same again but change the function to `InelasticDiffRotDiscreteCircle`

```
PlotPeakByLogValue(
    Input='irs26176_graphite002_red,i0;irs26176_graphite002_red,i1;irs26176_graphite002_red,i2;irs26176_graphite002_red,i3;irs26176_graphite002_red,i4;irs26176_graphite002_red,i5;irs26176_graphite002_red,i6;irs26176_graphite002_red,i7;irs26176_graphite002_red,i8;irs26176_graphite002_red,i9',
    OutputWorkspace='fit',
    Function='name=InelasticDiffSphere,Intensity=1,Radius=2,Diffusion=0.1,Shift=0'',
    FitType='Individual',
    PassWSIndexToFunction=True,
    CreateOutput=True,
    OutputCompositeMembers=True)
```
